tests/modules/nf-core/canu/nextflow.config

```diff
@@ -0,0 +1,9 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: CANU {
+	cpus = 1
```

Comment:
one more indent would make it look nicer

Reply:
I redid it anyway, hopefully it will look better